src/quantile.rs

```diff
@@ -278,6 +300,27 @@ where
     S: Data<Elem = A>,
     D: Dimension,
 {
+    fn argmin(&self) -> Option<D::Pattern>
+    where
+        A: PartialOrd,
+    {
+        let min = self.first()?;
```

Comment:
A stylistic suggestion: `min` is not the variable we use to keep track of the minimum - I'd rename it to something closer to what it does/is (`first_value`?).